src/applications/disability-benefits/all-claims/tests/pages/additionalDocuments.unit.spec.jsx

```diff
@@ -127,6 +128,14 @@ describe('526EZ document upload', () => {
       featureToggles: {},
     }));
 
+    // mock BDD
+    sessionStorage.setItem(
+      SAVED_SEPARATION_DATE,
+      moment()
+        .add(90, 'days')
+        .format('YYYY-MM-DD'),
+    );
+
```

Comment:
Nice! Kinda curious what was done in these tests before this, or if were testing around separation date at all. Might be worth a mental note or follow-up item to address as tech debt

Reply:
This view didn't have anything specific to BDD flow before and this seemed the easiest option to set up to test this view. The logic to trigger BDD can be found [here](https://github.com/department-of-veterans-affairs/vets-website/blob/58531-remove-alerts/src/applications/disability-benefits/all-claims/utils/index.jsx#L367).

Reply:
Gotcha, and sorry, I realize the thought I wrote out was kinda incoherent. Thanks for the context!